### ISSUE TITLE
overrides assignment not supported through javascript API

### ISSUE THREAD

**Issue by uxcourt** [BohemianCoding/SketchAPI#488]
My team would like to define layers in a common symbol library which label the UI components with the appropriate CSS class names to clarify when we deliver completed comps what elements have already been coded up.

I've created a plug in using skpm and node. Using the Sketch DevTools plug-in I was able to spelunk the object model iteratively to come up with an approach which finds the specific overrides I want to alter:

let selectedLayers = context.document.documentData().allSymbols();
var selectedCount = selectedLayers.length;
var x;
for(x=0;x<selectedCount;x++){
var y;
var oRideCount=selectedLayers[x].availableOverrides().length;
for(y=0;y<oRideCount;y++){
if(selectedLayers[x].availableOverrides()[y].affectedLayer().name()=="o.className"){
console.log(selectedLayers[x].availableOverrides()[y].overridePoint());
}
if(selectedLayers[x].availableOverrides()[y].affectedLayer().name()=="o.color"){
console.log(selectedLayers[x]);
} 
}

As you can see, I'm testing for two specific affectedLayer names, then I want to change the style override on those symbols only. So far, I haven't found an "overrrides" property or method which allows me to write the new value.

lots of other samples document what seems like a trivial operation:
just assign a new NSMutableDictionary to overrides, no need to call any method
in other words, 
instance.overrides=NSMutableDictionary

The problem is, through the javascript API, I can't find any object within 'context' that supports an overrides method or property. All of the other published interfaces for Symbol Instance https://developer.sketch.com/reference/api/#symbol-instance return 'undefined'.

**Comment by mathieudutour** [comment]
that's because `context` isn't part of the JS API.

If you look at the documentation, you will find that to get the document, you need:
```
const sketch = require('sketch')

const document = sketch.getSelectedDocument()
```

From there you can get the symbols, the overrides, etc. as shown in the documentation
